csrc/mmdeploy/backend_ops/tensorrt/common_impl/nms/allClassNMS.cu

```diff
@@ -66,17 +68,19 @@ __device__ float jaccardOverlap(const Bbox<T_BBOX> &bbox1, const Bbox<T_BBOX> &b
 /********** new NMS for only score and index array **********/
 
 template <typename T_SCORE, typename T_BBOX, int TSIZE>
-__global__ void allClassNMS_kernel(const int num, const int num_classes,
-                                   const int num_preds_per_class, const int top_k,
-                                   const float nms_threshold, const bool share_location,
-                                   const bool isNormalized,
-                                   T_BBOX *bbox_data,  // bbox_data should be float to preserve
-                                                       // location information
-                                   T_SCORE *beforeNMS_scores, int *beforeNMS_index_array,
-                                   T_SCORE *afterNMS_scores, int *afterNMS_index_array,
-                                   bool flipXY = false) {
+__global__ void
+#if __CUDA_ARCH__ == 620 || __CUDA_ARCH__ == 530
+__launch_bounds__(512)
+#endif
+    allClassNMS_kernel(const int num, const int num_classes, const int num_preds_per_class,
```

Comment:
fault indent

Reply:
It is formated by clang-format, any advice?

Reply:
use comment "// clang-format off" and "// clang-format on". Here is an example: https://github.com/open-mmlab/mmdeploy/blob/85320df2b46ed9a4752f68767b7deed4efc866d9/csrc/mmdeploy/apis/c/mmdeploy/model.cpp#L3